smart_proxy.gemspec

```
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'json'
   s.add_dependency 'logging'
   s.add_dependency 'rack', '>= 1.3'
+  s.add_dependency 'rexml', '~> 3.4'
```

If we leave it at 3.2 we can match what's bundled in Ruby 3.0: https://stdgems.org/rexml/
```suggestion
  s.add_dependency 'rexml', '~> 3.2'
```